pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -454,6 +454,7 @@ private void searchForCandidateBroker(NamespaceBundle bundle,
             // The leader election service was not initialized yet. This can happen because the broker service is
             // initialized first and it might start receiving lookup requests before the leader election service is
             // fully initialized.
+            LOG.warn("Leader election service isn't initialized yet. Returning empty result to lookup.");
```

Comment:
Do we have some meaningful context for whom reads the logs?

Reply:
Fixed